ROMFS/px4fmu_common/init.d/airframes/4250_generic_quad_250

```diff
@@ -1,32 +1,32 @@
 #!/bin/sh
 #
-# @name Generic 250 Racer
+# @name Generic Quadcopter (250mm)
```

Comment:
This is specifically for racers, can you keep it in the name?

Reply:
Done. Thanks!